CMake/mfi_defs.cmake

```diff
@@ -0,0 +1,23 @@
+#SDL Joystick axis mapping (circle-pad)
```

Comment:
Imo those changes should be defined during the cmake step:

```
cmake .. -DJOY_AXIS_LEFTX=0
...
```
Maybe adding this to the wiki page would be a good idea. But let's see what others think.